src/Agent/NewRelic/Profiler/ConfigurationTest/MockSystemCalls.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include <functional>
+#include <string>
+#include <memory>
+#include <unordered_map>
+#include "../Profiler/SystemCalls.h"
```

Comment:
If you want this to build on Linux, you will need to conditionally include the UnixSystemCalls.

Reply:
I didn't realize they built at all on Linux. Updated.